<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_Mike_on">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>460</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 52, 197);
font: 75 13pt &quot;Ubuntu Mono&quot;;</string>
   </property>
   <property name="text">
    <string>Mic On</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_User_Name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>사용자 명</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Output">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>580</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 255, 96);
font: 75 13pt &quot;Ubuntu Mono&quot;;</string>
   </property>
   <property name="text">
    <string>음성 출력</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_Model">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>550</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>F</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Default</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Label_User_Text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>마이크에서 입력받은 말을 텍스트로 출력</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Model_Text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>580</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>모델에서 생성한 텍스트 출력</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cam_window">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>451</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>web_cam</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
